packages/calcite-components/src/components/table/table.e2e.ts

```diff
@@ -1106,13 +1107,16 @@ describe("selection modes", () => {
     expect(await element.getProperty("selectedItems")).toHaveLength(1);
     await selectedItemAsserter([row1.id]);
 
-    await page.$eval("calcite-table", () => {
-      const table = document.querySelector("calcite-table");
-      const pagination = table.shadowRoot.querySelector("calcite-pagination");
-      const button = pagination.shadowRoot.querySelector<HTMLButtonElement>("button.page:nth-of-type(2)");
+    await page.$eval(
+      "calcite-table",
+      (table, PAGINATION_CSS) => {
+        const pagination = table.shadowRoot.querySelector("calcite-pagination");
+        const button = pagination.shadowRoot.querySelectorAll<HTMLButtonElement>(`.${PAGINATION_CSS.page}`)[1];
```

Comment:
@macandcheese @jcfranco the table tests are going into implementation details of the pagination component. Should we refactor these tests to use the correct pagination API rather than clicking internal elements?

Reply:
I think we should. Excellent suggestion!

Reply:
https://github.com/Esri/calcite-design-system/issues/9483